backends/client.go

```diff
@@ -4,6 +4,7 @@ type Client interface {
 	Load() ([]byte, error)
 	Save([]byte) error
 	Backup() error
+	SafeLocation() string
```

Comment:
I don't feel like this belongs on the backend client interface; it feels specific to the disk backend.